### PR TITLE
Fix #345

### LINKED ISSUE #345: check_ups: perfdata is invalid
The check_ups plugin provides voltage data using "mV" units (e.g. `voltage=218800mV`) which is not an allowed UOM (https://www.monitoring-plugins.org/doc/guidelines.html#AEN201).

The same applies to the temperature units "degC" and "degF" (though our UPS does not provide temperature readings so we are not getting errors because of that).

Moreover, warning and critical values are always multiplied by 1000, which seems to be incorrect in case of percents and temperatures.

### PR DESCRIPTION
**Pull request by gino0631**
Fixes problems described in #345.

### CODE REVIEW

**Comment by Madlohe** [comment]
Hi, @gino0631, thanks for the contribution!

Changing the output for temperatures is good, I'll merge that in.

With regard to the disallowed units, did you have difficulty integrating the plugin with any other software? I talked this over with the rest of the team, and we weren't able to think of anything that strictly enforced the allowed units.

**Comment by gino0631** [comment]
@Madlohe thanks for taking a look!

Yes, I implemented all these fixes because some performance data were missing in our setup (which is Icinga 2 + Graphite + Grafana), and there were messages in logs complaining about the units.

If some systems do not care and do work well with non-standard units, I think the plugin could have an "extended" mode, activated with a switch, to provide these units (if that's really important), but the operation should be according to the specs by default IMHO.

File: plugins/check_ups.c [hunk 1]
```diff
@@ -615,6 +620,8 @@ print_help (void)
   printf ("    %s\n", _("Name of UPS"));
   printf (" %s\n", "-T, --temperature");
   printf ("    %s\n", _("Output of temperatures in Celsius"));
+  printf (" %s\n", "-e, --extended-units");
+  printf ("    %s\n", _("Allow nonstandard units in performance data (used for voltage an temperatures)."));
```
Comment (gino0631) [inline plugins/check_ups.c]: @Madlohe an -> and; "." at the end is probably not necessary (not used for other options).

Comment (Madlohe) [reply plugins/check_ups.c]: Good catch, I've committed the changes to maint.

**Pull request merged**